test/physics/em/LivermorePE.test.cc

```diff
@@ -17,7 +17,7 @@
 #include "base/Range.hh"
 #include "comm/Device.hh"
 #include "io/AtomicRelaxationReader.hh"
-#include "io/ImportPhysicsTable.hh"
+#include "io/detail//ImportPhysicsTable.hh"
```

Comment:
Typo with the double-slash—but this should be reverted once these `Import` classes get moved back out of detail.